internal/distro/fedora/distro_test.go

```diff
@@ -575,7 +587,7 @@ func TestDistro_TestRootMountPoint(t *testing.T) {
 			_, err := imgType.Manifest(bp.Customizations, distro.ImageOptions{}, nil, testPackageSpecSets, nil, 0)
 			if imgTypeName == "iot-commit" || imgTypeName == "iot-container" || imgTypeName == "iot-raw-image" {
 				assert.EqualError(t, err, "Custom mountpoints are not supported for ostree types")
-			} else if imgTypeName == "iot-installer" || imgTypeName == "image-installer" {
+			} else if imgTypeName == "iot-installer" || imgTypeName == "image-installer" || imgTypeName == "minimal-raw" {
```

Comment:
Drop this change.  `minimal-raw` supports custom filesystems now and should be tested.